src/components/hosts/HardwareStatus.tsx

```diff
@@ -19,8 +20,18 @@ const HardwareStatus: React.FC<HardwareStatusProps> = (props) => {
     props.validationsInfo,
     'baremetal-discovery',
   );
+  const sublabel = getFailingClusterWizardSoftValidationIds(validationsInfo, 'baremetal-discovery')
+    .length
+    ? 'Some validations failed'
+    : undefined;
+
   return (
-    <HostStatus {...props} statusOverride={hardwareStatus} validationsInfo={validationsInfo} />
+    <HostStatus
```

Comment:
I think it is not necessary to calculate the sublabel here. Since we're passing the `validationsInfo` which contains only validations relevant for the given step, to determine the sublabel a `HostStatus` component can calculate the sublabel by flattening the validationsInfo values and checking if only failing validations are the 'soft' ones (very similar to `checkHostValidations` function.

Reply:
The answer depends on complexity of the calculation (discussion bellow). If the definition of soft validations is simplified, then yes, we can calculate that sublabel here along the two ramaining cases. But if it is dependent on the context (I mean wizard step), then I would keep `HostStatus` generic.